java/ql/test/experimental/query-tests/security/CWE-555/messages.properties

```diff
@@ -0,0 +1,8 @@
+prompt.username=Username
```

Comment:
Maybe add a comment here are the beginning to indicate that these should not be matched:
```suggestion
# GOOD: UI display messages; not credentials
prompt.username=Username
```

Reply:
Added